pkg/apisix/cache/cache.go

```diff
@@ -35,6 +35,8 @@ type Cache interface {
 	InsertGlobalRule(*v1.GlobalRule) error
 	// InsertConsumer adds or updates consumer to cache.
 	InsertConsumer(*v1.Consumer) error
+	// InsertSchema adds or updates schema to cache.
+	InsertSchema(*v1.Schema) error
```

Comment:
Better to use `InsertPluginSchema`.

Reply:
`v1.Schema` is the schema of APISIX objects, including plugin, route, upstream, etc. So `InsertSchema` can be reused.